### PR TITLE
Added more links to screencast videos

### PR DESCRIPTION
**Pull request by javiereguiluz**
Lots of people love these video tutorials, so let's link to more of them in some important pages.

### CODE REVIEW

File: doctrine.rst [hunk 1]
```diff
@@ -895,3 +900,4 @@ Learn more
 .. _`newer utf8mb4 character set`: https://dev.mysql.com/doc/refman/5.5/en/charset-unicode-utf8mb4.html
 .. _`Transactions and Concurrency`: http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/transactions-and-concurrency.html
 .. _`limit of 767 bytes for the index key prefix`: https://dev.mysql.com/doc/refman/5.6/en/innodb-restrictions.html
+.. _`Doctrine screencast series`: https://knpuniversity.com/screencast/symfony-doctrine
```
Comment (weaverryan) [inline doctrine.rst]: For Symfony 3: https://knpuniversity.com/screencast/symfony3-doctrine

File: doctrine/associations.rst [hunk 2]
```diff
@@ -413,3 +419,4 @@ Doctrine's `Association Mapping Documentation`_.
 
 .. _`Association Mapping Documentation`: http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/association-mapping.html
 .. _`Association Updates`: http://docs.doctrine-project.org/en/latest/reference/unitofwork-associations.html
+.. _`Mastering Doctrine Relations`: https://knpuniversity.com/screencast/doctrine-relations
```
Comment (weaverryan) [inline doctrine/associations.rst]: https://knpuniversity.com/screencast/symfony3-doctrine-relations

File: forms.rst [hunk 3]
```diff
@@ -699,3 +704,4 @@ Learn more
 .. _`Symfony Form component`: https://github.com/symfony/form
 .. _`DateTime`: https://php.net/manual/en/class.datetime.php
 .. _`2.8 UPGRADE Log`: https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.8.md#form
+.. _`Symfony Forms screencast series`: https://knpuniversity.com/screencast/symfony-forms
```
Comment (weaverryan) [inline forms.rst]: https://knpuniversity.com/screencast/symfony3-forms

File: security.rst [hunk 4]
```diff
@@ -1320,3 +1325,4 @@ Other Security Related Topics
 .. _`frameworkextrabundle documentation`: https://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/index.html
 .. _`HWIOAuthBundle`: https://github.com/hwi/HWIOAuthBundle
 .. _`Symfony ACL bundle`: https://github.com/symfony/acl-bundle
+.. _`Symfony Security screencast series`: https://knpuniversity.com/screencast/symfony-security
```
Comment (weaverryan) [inline security.rst]: https://knpuniversity.com/screencast/symfony3-security

File: service_container.rst [hunk 5]
```diff
@@ -1180,3 +1185,4 @@ Learn more
 .. _`service-oriented architecture`: https://en.wikipedia.org/wiki/Service-oriented_architecture
 .. _`Symfony Standard Edition (version 3.3) services.yml`: https://github.com/symfony/symfony-standard/blob/3.3/app/config/services.yml
 .. _`glob pattern`: https://en.wikipedia.org/wiki/Glob_(programming)
+.. _`Symfony Fundamentals screencast series`: https://knpuniversity.com/screencast/symfony-fundamentals
```
Comment (weaverryan) [inline service_container.rst]: https://knpuniversity.com/screencast/symfony3-fundamentals

File: page_creation.rst [hunk 6]
```diff
@@ -18,10 +18,11 @@ two-step process:
    create a Symfony ``Response`` object, which can hold HTML content, a JSON
    string or even a binary file like an image or PDF.
 
-.. seealso::
+.. admonition:: Screencast
+    :class: screencast
 
     Do you prefer video tutorials? Check out the `Joyful Development with Symfony`_
-    screencast series from KnpUniversity.
+    screencast series.
```
Comment (weaverryan) [inline page_creation.rst]: We should update the URL of this one too :) https://knpuniversity.com/screencast/symfony3

File: setup.rst [hunk 7]
```diff
@@ -7,10 +7,11 @@ Installing & Setting up the Symfony Framework
 This article explains how to install Symfony in different ways and how to solve
 the most common issues that may appear during the installation process.
 
-.. seealso::
+.. admonition:: Screencast
+    :class: screencast
 
     Do you prefer video tutorials? Check out the `Joyful Development with Symfony`_
-    screencast series from KnpUniversity.
```
Comment (weaverryan) [inline setup.rst]: And we should update this URL

**Review by weaverryan — COMMENTED** [review]
I like it: these are the most direct links to the most popular screencasts, so that people who like screencasts know these are available.

I just added comments so we can target the Symfony 3 tutorials here on 3.4. Then we can make another PR to target the V4 tutorials on the 4 branch

Thanks!

**Comment by javiereguiluz** [comment]
I've updated all URLs (and will change them again when merging into 4.0). Thanks for reviewing this!

**Pull request closed without merging**